src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        int ballNumberBound = 100;
+        return new Ball(colorSupplier.getRandomColor(), new Random().nextInt(ballNumberBound));

```

Comment:
100 should be a constant field

also, we can declare new ColorSupplier() and new Random() as fields. so that we don't create a new object each time the method is called